### ISSUE TITLE
playing list can not stop when I used jazzradio at second time

### ISSUE THREAD

**Issue by yuutayamada** [syohex/emacs-jazzradio#1]
Hello, I tried this package with windows.el and I encountered an issue which can not stop 
playing jazz when I re-entered _jazzradio_ buffer by jazzradio function.
It seems jazzradio function refleshed existing process data.
I could solve this problem by below code, but you may have good idea than me.
So I just conformed here:)

Thanks.

``` lisp
(defun jazzradio ()
  "Show channels menu."
  (interactive)
  (let* ((buf (get-buffer "*jazzradio*")))
    (if buf
        (switch-to-buffer buf)
      (setq buf (get-buffer-create "*jazzradio*"))
      (with-current-buffer buf
        (jazzradio--refresh)
        (jazzradio-menu-mode)
        (tabulated-list-print t)
        (switch-to-buffer buf)))))
```

**Comment by syohex** [comment]
Thanks for information

**Comment by syohex** [comment]
I have fixed at #2. Please check latest version.

**Comment by yuutayamada** [comment]
It worked perfectly fine! thank you!
